src/components/Card.tsx

```diff
@@ -89,15 +97,14 @@ const cardContainer = (
       'py-5': paddingType === 'normal',
       'py-8': typeof paddingType === 'undefined',
     }),
-    width(
-      thin ? 'sm:!w-thin-card' : 'sm:w-card',
-      thin ? 'xs:w-thin-mobile' : 'w-mobile-card',
-      { 'w-32': thin }
-    ),
+    useAppStyles ? undefined : thin ? thinWidthStyles : defaultWidthStyles,
```

Comment:
Can it be simplified and moved to the `const` variable? Like:
```
const widthStyle = () => {
  if (useAppStyles) return undefined
  return thin ? thinWidthStyles : defaultWidthStyles
}
```
I think it will be more convenient to use